partialWaveFit/fitResult.cc

```diff
@@ -343,6 +342,14 @@ fitResult::fill(const fitResult& result)
 	_hasHessian             = result.hasHessian();
 }
 
+multibinCenterType fitResult::multibinCenter() const
+{
+	multibinCenterType multibinCenter;
+	for (binningMapType::const_iterator it = _binningMap.begin(); it != _binningMap.end(); ++it) {
+		multibinCenter[it->first] = 0.5 * (it->second.first + it->second.second);
+	}
+	return multibinCenter;
+}
```

Comment:
I just wanted to implement a general function that calculates the center of the multibin. 
At the moment, is is anyway only used for the mass, so it could be removed, but if we later have another binning variable (for what ever reason), a general bin-center function could be useful. 
What do you think? @suhlatwork